analysis/analysis.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+
+from matplotlib.pyplot import figure, show
+import datetime
+import dateutil.parser
+import json
+import numpy as npy
+import os
+import pprint
+import sys
+import urllib
+#import pylab as plt
+
+if (len(sys.argv) == 1 or sys.argv[1] == "-h"):
+    print "Usage: " + sys.argv[0] + " json_output [csv_output]"
+    print
+    print " `-> Print the analysis as json into the json_output file, and,"
+    print "     if another file is given, write as CSV the informations concerning"
+    print "     the maximum amount of WiFi devices measured by day, by captor, into"
+    print "     the csv_output file."
+    print
+    exit()
+
+secret_json = os.path.dirname(os.path.abspath(__file__)) + "/../PRIVATE.json"
+with open(secret_json) as configuration_json:
+    configuration = json.load(configuration_json)
+
+pp = pprint.PrettyPrinter(indent=4)
+allinfos_url = configuration["data_source"] + "/allPlacesInfos"
+allinfos = json.loads(urllib.urlopen(allinfos_url).read())
+allcaptors = []
+base = datetime.datetime.today()
+
+opening_hours_url = configuration["data_source"] + "/sensor/getAll?s=" + configuration["secret"]
+opening_hours = json.loads(urllib.urlopen(opening_hours_url).read())
+
+csv_output = open("/dev/null", "w+")
+if (len(sys.argv) > 2):
+    csv_output = open(sys.argv[2], "w+")
+
+def nb_measures_expected(captor_id, hour, month, day):
+    hour += 1
+    if (month > 3) or (month == 3 and day > 27):
+        hour += 1
+    for captor in opening_hours:
```

Comment:
The correct word for "Capteur" is "Sensor".
A more understandable line here would be:
`for sensor in sensors:`